### PR TITLE
Code Cleanups & Reorganizing

### PR DESCRIPTION
**Pull request by dpogue**
Technically this is able to be merged now, but it's a first-pass at cleaning some stuff up. There's lots more work to do. It might be easier to merge in smaller chunks though rather than letting it all pile up.

Each commit should be relatively self-explanatory.

### CODE REVIEW

**Comment by zrax** [comment]
I think I might extract your loggable stuff into its own PR with some removals of other plString::Format-deprecation stuff...

**Comment by dpogue** [comment]
Hold this until #497 is merged and then I'll rebase on top of it.

**Comment by dpogue** [comment]
rebased

File: Sources/Plasma/PubUtilLib/plPipeline/DX/plDXPipeline.cpp [hunk 1]
```diff
@@ -9132,25 +9115,6 @@ void plDXPipeline::SetDepth(float hither, float yon)
     IGetViewTransform().SetDepth(hither, yon);
 }
 
-//// ISavageYonHack ///////////////////////////////////////////////////////////
-//  Corrects the yon for the *#(&$*#&$(*& Savage4 chipset (ex. Diamond Stealth
-//  III S540). Let's just say this card SUCKS.
-// Obsolete since we don't support the Savage4 chipset any more.
```
Comment (Hoikas) [inline Sources/Plasma/PubUtilLib/plPipeline/DX/plDXPipeline.cpp]: :trollface:

File: Sources/Plasma/PubUtilLib/plPipeline/DX/plDXPipeline.cpp [hunk 2]
```diff
@@ -11973,32 +11777,6 @@ const char  *plDXPipeline::IGetDXFormatName( D3DFORMAT format )
     }
 }
 
-//// IFPUCheck ////////////////////////////////////////////////////////////////
-//  Checks the FPU to make sure it's in the right mode
-// This should return wSave to allow it to be restored after rendering.
-// This is obsolete as of DX8
```
Comment (Hoikas) [inline Sources/Plasma/PubUtilLib/plPipeline/DX/plDXPipeline.cpp]: :trollface:

**Comment by dpogue** [comment]
Updated to address comments.

**Pull request merged**